Frontend/MoptPaymentPayone/Bootstrap.php

```diff
@@ -541,6 +541,11 @@ protected function createDatabase()
             $this->getInstallHelper()->fcExtendConfigRatepayInstallmentModeDataTable();
         }
 
+        // config option for AmazaonPay Pacstation allow
+        if (!$this->getInstallHelper()->fcPacStationAllowedForAmazonPayExtensionExist()) {
```

Comment:
Please rewrite all occurrences of "Pac" to say "Pack", this will make it clearer what this is about.